### PR TITLE
completed the alert and button component

### PR DESCRIPTION
**Pull request by yogesh-**

### CODE REVIEW

File: alert_button.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
+
+/* Alert */
+
+.alert{
+height: 4rem;
+/* width: 100%; */
```
Comment (tusharkandpal) [inline alert_button.css]: Remove this commented code

File: alert_button.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    filter: brightness(90%);
+}
+
+.btn-primary{
+    background: lightskyblue;
+    color: black;
+}
+
+.btn-secondary{
+color: black;
```
Comment (tusharkandpal) [inline alert_button.css:70]: Format these classes

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
Would suggest you to provide a brief description for any PR
Second, split your doc html into various components html since as code will grow, it will be much easier to make changes in.

**Comment by tanishk15sharma** [comment]
change suggestions given by @tusharkandpal and then you are good to go.

**Review by tusharkandpal — APPROVED** [review]
Seems ok ! Can merge

File: alert_button.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+.btn-error{
+background: red;
+color: white;
+}
+
+.btn-success{
+    background: green;
+    color: white;
+}
```
Comment (stutisk) [inline alert_button.css:82]: Instead of repeating the color you can try giving it a separate class

File: alert_button.css [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+.alert{
+height: 4rem;
+/* width: 100%; */
+color: white;
+margin: 2rem 2rem;
+border-radius: 0.5rem;
+padding: 1rem 2rem;
+font-size: 1.5rem;
+}
+
```
Comment (stutisk) [inline alert_button.css:12]: Try removing these extra spaces

**Review by stutisk — COMMENTED** [review]
Else looks good , just try removing the extra spaces , and beautifying your code

File: alert_button.css (hunk 2 above)
Comment (yogesh-) [reply alert_button.css:70]: done

File: alert_button.css (hunk 3 above)
Comment (yogesh-) [reply alert_button.css:82]: okay

**Pull request merged**